.jshintignore

```diff
@@ -0,0 +1,2 @@
+assets/javascripts/gridster/jquery.gridster.min.js
+assets/javascripts/jquery.knob.min.js
```

Comment:
I tried to put them in `vendor` but Sprockets did not like that. I couldn't get it to work easily, so I think we should make this a separate exercise in a separate PR.